pkg/api/apis/operators/installplan_types.go

```diff
@@ -354,6 +354,9 @@ type StepResource struct {
 	Kind                   string
 	Name                   string
 	Manifest               string
+	// BundleSecret is a one-off flag for handling secrets from a user bundle versus from the catalog source.
+	// Longer term StepResources will be refactored.
+	BundleSecret           bool
```

Comment:
Can we call out that this field is handled internally by OLM and should not be exposed by the API?

Reply:
Sure!